legion/engine/physics/physicsconstants.hpp

```diff
@@ -12,9 +12,9 @@ namespace legion::physics::constants
 
 	static constexpr float faceToFacePenetrationBias = 0.005f;
 
-	static constexpr float faceToEdgePenetrationBias = 0.01f;
+	static constexpr float faceToEdgePenetrationBias = 0.05f;
 
-	static constexpr float baumgarteCoefficient = 0.3f;
+	static constexpr float baumgarteCoefficient = 0.50f;
```

Comment:
It concers me that the coefficient and bias decided to change, but I'll allow it